.github/workflows/runtime-dockerhub.yml

```diff
@@ -0,0 +1,29 @@
+name: Build docker image and deploy
+
+on:
+  push:
+    branches: main
+    paths:
+      - environments/**
+  pull_request: # only to test while developing
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        image: [runtime_0-0-3_py39]
```

Comment:
Currently this is pushing to `coiled/runtime_0-0-3_py39:latest` on docker hub. However, I think `coiled/coiled-runtime-py39:0.0.3` uses a more conventional name. @ncclementi thoughts on using this naming convention instead?